src/components/rich_text_char_limit_1000/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Wysiwyg from '@/components/wysiwyg'
+import { ParagraphRichTextCharLimit1000 } from '@/types/paragraph'
+
+interface WysiwygProps {
+  field_wysiwyg: ParagraphRichTextCharLimit1000
+  id: number
+  className?: string
+}
+
+function RichTextCharLimit1000(
+  wysiwygDataService: WysiwygProps,
+  className: string
+): JSX.Element {
+  return (
+    <Wysiwyg
+      id={wysiwygDataService.id}
+      html={wysiwygDataService.field_wysiwyg.processed}
+      className={className}
```

Comment:
This is a little strange so I'm going to look for ways to destructure  the data so it's more clear

Reply:
fixed:

```
export const RichTextCharLimit1000 = ({
  id,
  html,
  className,
}: RichTextCharLimit1000Props): JSX.Element => {
  return <Wysiwyg id={id} html={html} className={className} />
}
```